README.md

```diff
@@ -58,40 +58,13 @@ editor.setHtmlContent('hello world');
 
-Unlike the example which does not specify the version, for production use, we recommend to use a fixed major version number to avoid breaking changes. The changelog can be seen [on Github](https://github.com/lblod/frontend-embeddable-notule-editor/releases), any update with breaking changes will have a higher version number. For example, to have the latest version of the v3 release, use the following imports:
+Unlike the example which does not specify the version, for production use, we recommend to use a fixed major version number to avoid breaking changes. The changelog can be seen [on Github](https://github.com/lblod/frontend-embeddable-notule-editor/releases), any update with breaking changes will have a higher version number. For example, to have the latest version of the v3 release, use the following import:
 
 > [!NOTE]
-> It's important to use the same version string (it can be any semver range or tag supported by unpkg).
+> the version string can be any semver range or tag supported by unpkg.
 
 ```html
-<link rel="stylesheet" href="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/vendor.css" />
-<link rel="stylesheet" href="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/@lblod/embeddable-say-editor.css" />
-<script src="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/vendor.js"></script>
-<script src="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/@lblod/embeddable-say-editor.js"></script>
-<script src="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/@lblod/embeddable-say-editor-app.js"></script>
-```
-
-### Building the sources yourself
-
-In order to build the JavaScript and CSS sources of this repository you will need `ember-cli` installed (more info at the [Development section](#development-of-@lblod/embeddable-say-editor)), then execute the following:
-
-```bash
-git clone https://github.com/lblod/frontend-embeddable-notule-editor.git
-cd frontend-embeddable-notule-editor
-npm install
-npm run build
-```
-
-In the 'dist' folder structure, two CSS files and three JavaScript files will have been generated under assets. These are the files to use, as demonstrated in the [Code Example](#basic-example-the-editor-in-an-html-file). Note that the fingerprints of your files may vary. Other files in the dist folder can be ignored at this point, as they are not needed.
-
-```bash
-dist
-└── assets
-    ├── @lblod/embeddable-say-editor-app.js
-    ├── @lblod/embeddable-say-editor.css
-    ├── @lblod/embeddable-say-editor.js
-    ├── vendor.css # currently empty and not needed
-    └── vendor.js
+<script src="https://unpkg.com/@lblod/embeddable-say-editor@3.1.x"></script>
```

Comment:
I have a slight preference for pushing people (lightly) to just fix the major number as minor versions should be compatible and it avoids people being stuck on old versions...

Reply:
I pinned it cause at the time the 3.2.0 release had a broken npm package build

Reply:
that's now resolved so I set it to ^3.2.1